src/main/webapp/app/course/manage/course-management.module.ts

```diff
@@ -124,6 +125,7 @@ import { ArtemisMarkdownEditorModule } from 'app/shared/markdown-editor/markdown
         DetailModule,
         SubmissionResultStatusModule,
         ArtemisMarkdownEditorModule,
+        ArtemisFAQModule,
```

Comment:
There shouldn't really be a need for us to define modules anymore. Could you directly import what you need and remove the module?

Reply:
Refatored it to adress the new client guidelines.